lib/mayaUsd/render/vp2RenderDelegate/mayaPrimCommon.cpp

```diff
@@ -687,13 +687,74 @@ void MayaUsdRPrim::_UpdatePrimvarSourcesGeneric(
     }
 }
 
-void MayaUsdRPrim::_SyncDisplayLayerModes(const HdRprim&
 #ifdef MAYA_HAS_DISPLAY_LAYER_API
-                                              refThis
+void MayaUsdRPrim::_PopulateDisplayLayerModes(
```

Comment:
Move some code to create a new function _PopulateDisplayLayerModes to be used elsewhere. Basically just a refactoring